### PR TITLE
[WIP] Add a preview step before recording the appointment summary

### PR DESCRIPTION
**Pull request by stevenwilkin**
Feedback on the cukes would be useful

### CODE REVIEW

File: features/support/pages/record_of_guidance_preview_page.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class RecordOfGuidancePreviewPage < SitePrism::Page
+  set_url_matcher %r{/appointment_summaries/preview}
+
+  element :name, '.name'
```
Comment (gjvis) [inline features/support/pages/record_of_guidance_preview_page.rb]: Is the lack of the `t-` class preference intentional?

Comment (stevenwilkin) [inline features/support/pages/record_of_guidance_preview_page.rb]: The HTML view of the `OutputDocument` is rendered in the page which includes a paragraph with a class of `name`

**Comment by stevenwilkin** [comment]
Cuke updated

**Comment by stevenwilkin** [comment]
Closing this in favour of another PR that combines the confirmation page with the updated HTML view of the output document

**Pull request closed without merging**